server/shd.py

```diff
@@ -0,0 +1,64 @@
+"""
+Starts Gluster Brick(fsd) process
+"""
+import os
+import uuid
+import sys
+import json
+import logging
+import requests
+
+from jinja2 import Template
+
+from kadalulib import logf, send_analytics_tracker
+
+
+VOLFILES_DIR = "/kadalu/volfiles"
+TEMPLATES_DIR = "/kadalu/templates"
+VOLINFO_DIR = "/var/lib/gluster"
+
+
+def generate_shd_volfile(volfile_path, volname):
+    """Generate Client Volfile for Glusterfs Volume"""
+    info_file_path = os.path.join(VOLINFO_DIR, "%s.info" % volname)
+    data = {}
+    with open(info_file_path) as info_file:
+        data = json.load(info_file)
+
+    template_file_path = os.path.join(TEMPLATES_DIR, "Replica3.shd.vol.j2")
+    client_volfile = os.path.join(VOLFILES_DIR, "glustershd.vol")
+    content = ""
+    with open(template_file_path) as template_file:
+        content = template_file.read()
+
+    Template(content).stream(**data).dump(client_volfile)
+
+
+def start():
+    """
+    Start the Gluster Self-Heal Process
+    """
+    volname = os.environ["VOLUME"]
+
+    volfile_path = os.path.join(VOLFILES_DIR, "glustershd.vol")
+    generate_shd_volfile(volfile_path, volname)
+
+    # Send Analytics Tracker
+    # The information from this analytics is available for
+    # developers to understand and build project in a better way
+    send_analytics_tracker("server")
```

Comment:
considering this is another container in server pod, we can skip this IMO

Reply:
ack